test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -87,6 +88,7 @@ public class CompactRecordLogger {
           entry("EVENT", "EVNT"),
           entry("DECISION_REQUIREMENTS", "DRG"),
           entry("EVALUATION", "EVAL"),
+          entry("SIGNAL", "SIG"),
           entry("SIGNAL_SUBSCRIPTION", "SIG_SUB"));
```

Comment:
❌ Please swap these. The order matters. In the current order, records of type `SIGNAL_SUBSCRIPTION` would be abbreviated to `SIG_SUBSCRIPTION`  instead `SIG_SUB`.